pkgs/objective_c/ffigen_objc.yaml

```diff
@@ -10,6 +10,12 @@ headers:
 ffi-native:
 exclude-all-by-default: true
 generate-for-package-objective-c: true
+objc-min-target-version:
+  # See https://docs.flutter.dev/reference/supported-platforms.
+  ios:
+    min: 2.5.0
```

Comment:
`2.5.0`? Shouldn't this be the "Supported versions" or "Best effort versions" from the linked URI?

Reply:
Oops. Copypasta